uasc/message_test.go

```diff
@@ -22,11 +22,13 @@ func TestMessage(t *testing.T) {
 					cfg: &Config{
 						SecurityPolicyURI: "http://gopcua.example/OPCUA/SecurityPolicy#Foo",
 					},
-					requestID:       1,
-					sequenceNumber:  1,
+				}
+				instance := &channelInstance{
+					sc: s,
+					sequenceNumber:  0,
```

Comment:
has the logic changed for getting the sequence numbers so that we can start at 0 now? I'm all for it, just curious.

Reply:
There's a atomic get/increment which returns the _new_ value. hence starts with 1.